roles/mailserver/tasks/dmarc.yml

```diff
@@ -1,5 +1,11 @@
-- name: Install OpenDMARC milter
-  apt: pkg=opendmarc state=installed update_cache=yes
+- name: Install OpenDMARC milter and related packages
+  apt: pkg={{item}} state=installed
+  with_items:
+    - mysql-server
+    - opendmarc
+    - python-mysqldb
```

Comment:
This `python-mysqldb` dependency is actually for the `mysql_db` and `mysql_user` _Ansible_ modules, not for OpenDMARC itself.  I'd also see maybe installing it in a different task (maybe over here in [common/main.yml][1]?), with a comment explaining its use.

[1]: https://github.com/sovereign/sovereign/blob/master/roles/common/tasks/main.yml#L19-L42